components/backdrop/backdrop.js

```diff
@@ -1,16 +1,19 @@
 import '../colors/colors.js';
+import '../../helpers/queueMicrotask.js';
 import { css, html, LitElement } from 'lit-element/lit-element.js';
-import { getComposedChildren, getComposedParent } from '../../helpers/dom.js';
+import { getComposedChildren, getComposedParent, isVisible } from '../../helpers/dom.js';
 import { getUniqueId } from '../../helpers/uniqueId.js';
 
+const reduceMotion = matchMedia('(prefers-reduced-motion: reduce)').matches;
+
 const scrollKeys = [];
 let scrollOverflow = null;
 
 class Backdrop extends LitElement {
 
 	static get properties() {
 		return {
-			shown: { type: Boolean },
+			shown: { type: Boolean, attribute: true },
```

Comment:
Is this needed? I thought observers were created automatically and you only needed to set this if you wanted to rename the attribute or not have an observer.

Reply:
I just tried this out again and it appear to work without... will remove.  I must have added while having some trouble and thought that it was needed.